include/LLGL/Container/StringView.h

```diff
@@ -114,7 +114,7 @@ class LLGL_EXPORT BasicStringView
 
         /**
         \brief Returns the raw pointer of this string view.
-        \remarks This does not necessarily point to a NUL-terminated string like \c std::string does.
+        \remarks This does not necessarily point to a NUL-terminated string like \c string does.
```

Comment:
Regarding the changes in the comments, I don't think these will be necessary. I presume you used a _find & replace_ function, but since these comments make an example under the assumption of using the C++ STL by default, I think we should keep them.